### PR TITLE
Upgrade base image to bionic

### PR DESCRIPTION
**Pull request by howardjohn**
This shaves about 50mb off images

Before we merge this we would need to switch to official repo of course, just didn't want to push until approved

This is not intended for 1.3 backport

### CODE REVIEW

**Comment by howardjohn** [comment]
ah it requested review from the whole world since I was testing with my own hub rather than pushing to official hub before approval. sorry

**Comment by howardjohn** [comment]
@sdake mind taking a look at this once the 1.3 release calms down?

File: galley/docker/Dockerfile.galley [hunk 1]
```diff
@@ -5,7 +5,7 @@ ARG BASE_DISTRIBUTION=default
 ARG BASE_VERSION=latest
 
 # The following section is used as base image if BASE_DISTRIBUTION=default
-FROM docker.io/istio/base:${BASE_VERSION} as default
+FROM howardjohn/base:${BASE_VERSION} as default
```
Comment (ozevren) [inline galley/docker/Dockerfile.galley]: howardjohn?

**Comment by howardjohn** [comment]
> Seems like there is some cleanup needed in the PR.

Oz, we don't have automation of building base images yet. As mentioned in the description

> Before we merge this we would need to switch to official repo of course, just didn't want to push until approved

**Comment by rlenglet** [comment]
`build-tools` is still using `ubuntu:xenial`. Do you plan to move that over as well?

**Comment by howardjohn** [comment]
@rlenglet Yes I think we can/should update that too, but I don't think there is any reason they must be one the same base

**Pull request merged**